src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1067,6 +1069,214 @@ public void clickRemoveMsqOptionLinkForNewQuestion(int optionIndex) {
+    }
+
+    private String getMsqMinSelectableChoicesCheckboxId(int qnNumber) {
+        return "msqEnableMinSelectableChoices-" + qnNumber;
+    }
+
+    private String getMsqMaxSelectableChoicesBoxId(int qnNumber) {
+        return "msqMaxSelectableChoices-" + qnNumber;
+    }
+
+    private String getMsqMinSelectableChoicesBoxId(int qnNumber) {
+        return "msqMinSelectableChoices-" + qnNumber;
+    }
+
+    public WebElement getMsqMaxSelectableChoicesElement(int qnNumber) {
+        String id = getMsqMaxSelectableChoicesCheckboxId(qnNumber);
+
+        return browser.driver.findElement(By.id(id));
+    }
+
+    private WebElement getMsqMinSelectableChoicesElement(int qnNumber) {
+        String id = getMsqMinSelectableChoicesCheckboxId(qnNumber);
+
+        return browser.driver.findElement(By.id(id));
+    }
+
+    public void toggleMsqMaxSelectableChoices(int qnNumber) {
+        click(getMsqMaxSelectableChoicesElement(qnNumber));
+    }
+
+    public void toggleMsqMinSelectableChoices(int qnNumber) {
+        click(getMsqMinSelectableChoicesElement(qnNumber));
+    }
+
+    public boolean isMsqMaxSelectableChoicesEnabled(int qnNumber) {
+        String id = getMsqMaxSelectableChoicesCheckboxId(qnNumber);
+
+        return browser.driver.findElement(By.id(id)).isSelected();
+    }
+
+    public boolean isMsqMinSelectableChoicesEnabled(int qnNumber) {
+        String id = getMsqMinSelectableChoicesCheckboxId(qnNumber);
+
+        return browser.driver.findElement(By.id(id)).isSelected();
+    }
+
+    private WebElement getMsqMaxSelectableChoicesBox(int qnNumber) {
+        return browser.driver.findElement(By.id(getMsqMaxSelectableChoicesBoxId(qnNumber)));
+    }
+
+    private WebElement getMsqMinSelectableChoicesBox(int qnNumber) {
+        return browser.driver.findElement(By.id(getMsqMinSelectableChoicesBoxId(qnNumber)));
+    }
+
+    public int getMinOfMsqMaxSelectableChoices(int qnNumber) {
+        return Integer.parseInt(getMsqMaxSelectableChoicesBox(qnNumber).getAttribute("min"));
+    }
+
+    public int getMaxOfMsqMaxSelectableChoices(int qnNumber) {
+        return Integer.parseInt(getMsqMaxSelectableChoicesBox(qnNumber).getAttribute("max"));
+    }
+
+    public int getMinOfMsqMinSelectableChoices(int qnNumber) {
+        return Integer.parseInt(getMsqMinSelectableChoicesBox(qnNumber).getAttribute("min"));
+    }
+
+    public int getMaxOfMsqMinSelectableChoices(int qnNumber) {
+        return Integer.parseInt(getMsqMinSelectableChoicesBox(qnNumber).getAttribute("max"));
+    }
+
+    public void setMsqMinSelectableChoices(int qnNumber, int value) {
+        assertTrue(isMsqMinSelectableChoicesEnabled(qnNumber));
+
+        WebElement inputBox = getMsqMinSelectableChoicesBox(qnNumber);
+        JavascriptExecutor exec = (JavascriptExecutor) browser.driver;
+        String id = inputBox.getAttribute("id");
+
+        exec.executeScript(String.format("$('#%s').val(%d);$('#%s').change();", id, value, id));
```

Comment:
You should not need to instantiate your own executor here. `executeScript` is available for all subclasses of `AppPage`.